python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -204,6 +204,14 @@ class RandomForestClassifier(Base):
     histogram-based algorithms to determine splits, rather than an exact
     count. You can tune the size of the histograms with the n_bins parameter.
 
+    **Known Limitations**: This is an early release of the cuML
+    Random Forest code. It contains a few known limitations:
+
+       * GPU-based inference is only supported if the model was trained
+         with 32-bit (float32) datatypes
```

Comment:
Maybe add a note that a CPU based float64 inference is provided for the time being?